### ISSUE TITLE
Unable to Quit/ terminate Visual Studio Code

### ISSUE THREAD

**Issue by tejaswipandava** [microsoft/vscode#95017]
I am trying to close the VS code but the close button doesn't work, it's doing nothing. There are no crash reports, nor any kind of stall in both the application as well as in windows. Checked in the task manager as well to see anything live not responding but it seems I am not getting any such issues.
Also after clicking close, I am still able to work on the opened application including debugging, creating new files and so on.

- VSCode Version: 1.44.0
- OS Version: Windows_NT x64 10.0.19041

Steps to Reproduce:

1. Open Visual Studio
2. Load Angular Project and use the terminal to run the app.
3. Wait for some time (1 hr) and try closing the vs code.

Does this issue occur when all extensions are disabled?: Yes/No
Yes.

**Comment by RMacfarlane** [comment]
@tejasrivastav When you're trying to close, is there something actively running in the terminal? Do you have the setting `terminal.integrated.confirmOnExit` set?

**Comment by tejaswipandava** [comment]
Yes, I was running a basic angular project which I didn't terminate. terminal.integrated.confirmOnExit is set to false.

Also, I have checked if we run a project and try to close the VS Code, It is closing. But if we leave the app running for over an hr and then try to close it fails.

**Comment by kshetline** [comment]
I've experienced this exact same problem, but on MacOS. What has been triggering the problem for me is trying to build a VSCode extension. If I start VSCode and never press F5 to debug the extension, everything is fine, and I can close and quit. Once I've used F5, however, even once, I must force-quit the application to get out of it. Unlike the OP, I don't have to wait an hour or so for this problem to occur.

Similarly to the OP, there are no crash reports or other signs of error. I have found one new `npm` process that's created when I launch the debugger, and killing that one process also allows be to close and quit, without having to do a forced quit.

Everything else is working fine except for the inability to easily close my current project, or quit the application.

I'm using VSCode 1.44.2, on MacOS 10.15.4.
